gallery/how_to/work_with_microtvm/micro_aot.py

```diff
@@ -0,0 +1,162 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""
+.. _tutorial-micro-AoT:
+
+microTVM Host-Driven AoT
+===========================
+**Authors**:
+`Mehrdad Hessar <https://github.com/mehrdadh>`_,
+`Alan MacDonald <https://github.com/alanmacd>`_
+
+This tutorial is showcasing microTVM host-driven AoT compilation with
+a TFLite model. This tutorial can be executed on a X86 CPU using C runtime (CRT)
+or on Zephyr plarform on a microcontroller that supports Zephyr platform.
+"""
+
+import numpy as np
+import pathlib
+import json
+import os
+
+import tvm
+from tvm import relay
+from tvm.relay.backend import Executor, Runtime
+from tvm.contrib.download import download_testdata
+
+######################################################################
+# Import a TFLite model
+# ---------------------
+#
+# To begin with, download and import a TFLite model from TinyMLPerf models.
```

Comment:
It's not actually  importing directly from the MLPerf Tiny bench, i.e. from:

https://github.com/mlcommons/tiny/tree/master/benchmark/training/keyword_spotting/trained_models

Should this benchmark be mentioned just as the original source of this model?

Also I think the current name for the bench is MLPerf Tiny, not TinyMLPerf, like in the text found in the READM.md here? https://github.com/mlcommons/tiny

Reply:
I added comment about the origin of the model and fixed MLPerf Tiny name.